event.html

```diff
@@ -59,34 +59,34 @@
     <div class="container">
         <div class="jumbotron event-jumbo mt-5 rounded">
 
-            <h2 class="p-2">
+            <h2 class="p-3">
                 SNBRN pool side at<br>phoenix hotel
             </h2>
             <img src="IMG/SNBRN.jpeg" class="h-50 mx-auto d-block">
-            <button type="button" class="btn btn-light col-2 m-4 d-flex justify-content-sm-center align-self-end">buy tickets</button>
+            <a class="btn btn-light col-2 m-4 p-3 d-flex justify-content-sm-center align-self-end text-uppercase "href="https://www.eventbrite.com/e/summertime-radness-dj-seinfeld-tickets-314960684887" target="_blank">buy tickets</a>
```

Comment:
As useful as Bootstrap is, I would encourage you to come back and actually write this CSS yourself. Reason being that when applying for a job employers are going to be more interested in your core CSS knowledge rather than how much you know Bootstrap. For example, you should be able to answer a lot of these questions: https://github.com/h5bp/Front-end-Developer-Interview-Questions/blob/main/src/questions/css-questions.md